bpf-metrics-exporter/README.md

```diff
@@ -14,24 +14,32 @@ to correlate process IDs -> containers -> k8s pods.
 
 The following metrics are exported:
 
-- `bpf_program_size_jitted_bytes`: The size of the BPF program in bytes.
-- `bpf_program_size_translated_bytes`: The size of the BPF program in bytes.
-- `bpf_program_mem_bytes`: The amount of memory used by the BPF program in bytes.
-- `bpf_program_verified_instructions_total`: The number of instructions in the BPF program.
-
-
-The following will be added pending: https://github.com/open-telemetry/opentelemetry-rust/issues/1242
-
-- `bpf_programs_total`: The number of BPF programs loaded into the kernel.
-
-Labels:
-
-- `id`: The ID of the BPF program
-- `name`: The name of the BPF program
-- `type`: The type of the BPF program
-- `tag`: The tag of the BPF program
-- `gpl_compatible`: Whether the BPF program is GPL compatible
-- `load_time`: The time the BPF program was loaded
+- `bpf_programs`: Information on each loaded BPF Program
+    - Labels:
+        - `id`: The ID of the BPF program
+        - `name`: The name of the BPF program
+        - `type`: The type of the BPF program as a readable string
+        - `tag`: The tag of the BPF program
+        - `gpl_compatible`: Whether the BPF program is GPL compatible
+        - `load_time`: The time the BPF program was loaded
+- `bpf_maps`: Information of each loaded BPF Map
+    - Labels:
+        - `id`: The ID of the BPF map
+        - `name`: The name of the BPF map
+        - `type`: The type of the BPF map as an `u32` which corresponds to the following [kernel enumeration](https://elixir.bootlin.com/linux/v6.6.3/source/include/uapi/linux/bpf.h#L906)
+        - `key_size`: The key size in bytes for the BPF map
+        - `value_size`: The value size for the BPF map
+        - `max_entries`: The maximum number of entries for the BPF map.
```

Comment:
Like `load_time` above, moving them into their own metrics could make more sense. It would allow to compute stats (e.g. total/avg/... of sizes across all bpf programs)

Reply:
Done!